package-orchestrator.sh

```diff
@@ -36,6 +36,7 @@ string=$(get_rpm_version_string)
 VERSION_VALUE=${string% *}
 RELEASE_VALUE=${string#* }
 PYTHON27_VALUE=0
```

Comment:
Maybe the `PYTHON27_VALUE` setting  (as long as `--with-python27`) should be removed?

Reply:
Python2.6 and python2.7 removed in
https://github.com/telefonicaid/orchestrator/pull/313/commits/16b2944a7c3b4264ba5547793ce7accf11f8fea1
https://github.com/telefonicaid/orchestrator/pull/313/commits/8b9b4e35038d9e0f58ea469c434e9ddbe01efd5d
since is not really supported now